src/InteractiveMars/Mars.tsx

```diff
@@ -0,0 +1,47 @@
+/* TO-DO 
```

Comment:
This is now 
a) A TypeScript file
b) Fixes the linting errors so we no longer have to disable es-lint.